### PR TITLE
refresh

### PR DESCRIPTION
**Pull request by siddhant3030**
- Refresh from DB

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -47,20 +46,28 @@ con <- dbConnect(
...
 location_query <- 'WITH events AS (SELECT title, type, category, status, description, location FROM "tabEvents" WHERE location IS NOT NULL and title IS NOT NULL),
 locations AS (SELECT name, CAST(latitude AS FLOAT8) AS lat, CAST(longitude AS FLOAT8) AS long, address, city, state, district FROM "tabLocations")
 SELECT * FROM events LEFT JOIN locations ON location = name'
-bqdata <- dbGetQuery(con, location_query)
 
-# List of distinct category Names
-category <- bqdata %>%
-  dplyr::select(category) %>%
-  distinct()
+get_data <- function() {
```
Comment (AkhileshNegi) [inline app.R]: @siddhant3030 rename to `frappe_data` which makes more sense

File: app.R [hunk 2]
```diff
@@ -47,20 +46,28 @@ con <- dbConnect(
...
+}
 
 # Reading all the data for Assembly level boundaries
 assembly_boundaries <- dbGetQuery(con, 'SELECT json FROM "boundaries" where id = 3')
-
 json_data <- assembly_boundaries$json
+
+
+# List of distinct category Names
+category <- get_data() %>%
```
Comment (AkhileshNegi) [inline app.R:67]: @siddhant3030 rename to `frappe_data`

File: app.R [hunk 3]
```diff
@@ -172,24 +179,23 @@ ui <- dashboardPage(
...
 server <- function(input, output, session) {
-
-  # This we need to auto connect the server.
-  session$allowReconnect(TRUE)
-
-
+  
+  bqdata <- reactivePoll(10000, session,
+                       checkFunc = check_for_update,
+                       valueFunc = get_data)
```
Comment (AkhileshNegi) [inline app.R]: @siddhant3030 rename to `frappe_data`

File: app.R [hunk 4]
```diff
@@ -268,37 +277,37 @@ server <- function(input, output, session) {
...
+                                           highlightOptions = highlightOptions(
+                                             weight = 2, color = "#9c4e57",
+                                             fillOpacity = 1, opacity = 1,
+                                             bringToFront = TRUE, sendToBack = TRUE
+                                           ),
+                                           pathOptions = pathOptions(
+                                             showMeasurements = TRUE,
+                                             measurementOptions =
+                                               measurePathOptions(imperial = TRUE)
+                                           ),
```
Comment (AkhileshNegi) [inline app.R]: seems formatting changes. Can you set the formatted in local in a way that does not reformat again

File: app.R [hunk 5]
```diff
@@ -268,37 +277,37 @@ server <- function(input, output, session) {
...
       ) %>%
-      hideGroup(group = "District boundaries") %>%
+      hideGroup(group = "district_boundaries") %>%
       # This is to add control layers on the map
       leaflet::addLayersControl(
         position = "bottomleft",
         baseGroups = c("Light"),
         overlayGroups =
-          c("District boundaries"),
+          c("district_boundaries"),
```
Comment (AkhileshNegi) [inline app.R]: @siddhant3030 any reason for this change. In UI button it was showing `district_boundaries` so I changed to `District boundaries` which lookes better UI wise

Comment (siddhant3030) [reply app.R]: See the logic above. If you change it here then you need to change the name of group which uses the same name.

Comment (siddhant3030) [reply app.R]: anyways I have updated it

File: app.R (hunk 4 above)
Comment (siddhant3030) [reply app.R]: done

**Pull request merged**